build.sh

```diff
@@ -0,0 +1,11 @@
+az acr login -n k8scc01covidacr
```

Comment:
Is this file getting commited?

Reply:
Had it for easy testing purposes, can remove it

Reply:
maybe just add this script in the readme? just instead of the actual file